build-setup.sh

```diff
@@ -50,8 +50,8 @@ do
             TOOLCHAIN=$1 ;;
         -ud | --unpinned-deps )
             USE_PINNED_DEPS=false ;;
-        --skip-validate)
-            SKIP_VALIDATE_FLAG=$1 ;;
+        --batch)
```

Comment:
You should maintain existing behavior for `--skip-validate` to avoid breaking existing flows (like CI, for example).

Reply:
I'll have to update the other flows but I can say that the skip validate is deprecated and remove it later (just to avoid bumping FireSim initially)

Reply:
What I mean is, leave in support for `--skip-validate`, and remove all references to this from the documentation.